src/main.rkt

```diff
@@ -57,7 +57,7 @@
   (define demo-port 8000)
   (define demo-public #f)
 
-  (define threads #f)
+  (define threads 1)
```

Comment:
Again I think `#f` for no threads and `1` for one thread should be separate (because even in "1 thread" mode you in fact spawn a number of places, even though only one of them is a worker thread.)